### PR TITLE
Move pieces

### PR DESCRIPTION
**Pull request by leamerluck**
@CodeFlowed @robgraphix1 I just noticed that my welcome page didn't get pulled up where I added the enter game buttons and list of available games :disappointed: I'll have to fix that! Oh, Nope... it's here! I'm losing my mind. LOL!
@Rails-Riders/owners This is ready to be shipped... although it isn't up to date with master. So there will be merge conflicts. I can work through those once we get a :+1: from someone.
UPDATE: I corrected the merge conflict. I forgot to pull down once Rob's is_obstructed was merged this morning, so I corrected that error. Ready to ship :smile:

### CODE REVIEW

File: app/controllers/pieces_controller.rb [hunk 1]
```diff
@@ -1,2 +1,24 @@
+require 'pry'
```
Comment (CodeFlowed) [inline app/controllers/pieces_controller.rb]: This is just for debugging, right?

Comment (robgraphix1) [inline app/controllers/pieces_controller.rb]: Hmmmm. That's weird. I Don't even know what that is actually. ??

Comment (leamerluck) [inline app/controllers/pieces_controller.rb]: @CodeFlowed @robgraphix1 yep its for debugging. My mentor introduced me to it.

**Comment by CodeFlowed** [comment]
Giving it a :+1:. Nice work, @leamerluck and @hugolmendez !

**Pull request merged**